server/controllers/chartsController/helpers.js

```diff
@@ -303,3 +331,84 @@ export const calculateSubjectVariableDayCount = (
 
   return subjectsDayDataAndVariable.length
 }
+
+export const addSubjectsToResultsFromFiltersMap = ({
+  subjectsInFiltersA,
+  subjectsInFiltersB,
+  mapForResultsFromFilters,
+}) => {
+  subjectsInFiltersA.filter((subjectData) => {
+    const foundSubject = subjectsInFiltersB.find(
+      ({ subject }) => subject === subjectData.subject
+    )
+    const isSubjectInMap = mapForResultsFromFilters?.has(subjectData.subject)
```

Comment:
is `subject` a string or an object or something else?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/280/commits/c30a7d514c931ad36f83916c440ff7812449f315